### ISSUE TITLE
Unable to use a Recipient field to send emails

### ISSUE THREAD

**Issue by jeffaglenn** [verbb/formie#721]
**Description**
Unable to use a Recipient field to send emails. According issue #280 this should be fixed? So I think I'm missing something simple.

I have my recipients field with multiple emails addresses:

<img width="1092" alt="Screen Shot 2022-01-17 at 1 36 12 PM" src="https://user-images.githubusercontent.com/34280463/149828586-1a6e1bf9-25df-440e-b6aa-0bd55a54f80b.png">

Then I have my Email Notification set up to pull from the field:

<img width="888" alt="Screen Shot 2022-01-17 at 1 39 34 PM" src="https://user-images.githubusercontent.com/34280463/149828751-289183df-f997-4776-8271-2c4b922df7a4.png">

When I submit a test it's pulling the Email Address ID instead of the email address:

<img width="995" alt="Screen Shot 2022-01-17 at 1 41 51 PM" src="https://user-images.githubusercontent.com/34280463/149829029-576970ee-2686-4888-901b-5e334981c007.png">

Am I doing something wrong?

**Steps to reproduce**
1. This is on a clean install of Craft and Formie

### Form settings
Please pick from the options below.

- Multi-page form:  No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.5.6
- Craft version: 3.7.29
- Multi-site: No

**Comment by engram-design** [comment]
Sorry about that. Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.6",
  "...": "..."
}
```

Then run `composer update`.

**Comment by jeffaglenn** [comment]
Yep, that worked. Thank you!

**Comment by engram-design** [comment]
Fixed in [1.5.7](https://github.com/verbb/formie/releases/tag/1.5.7)
